### DISCUSSION TITLE
true tone compatibilty

### DISCUSSION THREAD

**Discussion by michakluk** [MonitorControl/MonitorControl#841] [Ideas]
Hi, it would be lovely, if you could add true tone compatibility, to force some kind of filter according to the main macbook display.

**Comment by waydabber** [comment]
Hey @michakluk - the app is fully compatible with true-tone and macOS should apply the true-tone setting to the external display automatically (unless it is registered as a TV) if the lid is open and the built-in macbook display is active. At least this is how it works on M1 (I am not sure about various Intel models).
